### ISSUE TITLE
Error: similarity matrix for tuple-matrix in python

### ISSUE THREAD

**Issue by cinaljess** [ismms-himc/clustergrammer2#3]
Hello and thank you for the great tool.

I am using the Python version of clustergrammer2, specifically in jupyter notebook, and am unable to generate a sim_mat from my dataframe. The widget visualization of the same dataframe otherwise works when running 

`net.load_df(filtered_df)`
`net.cluster()`
`net.widget()`

The code that fails is:
`net.load_df(filtered_df)`
`net.cluster(sim_mat=True)`

Stack trace:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-30-504a038db812> in <module>
      1 net.load_df(filtered_df)
----> 2 net.cluster(sim_mat=True)

~/anaconda2/envs/sd2py3/lib/python3.6/site-packages/clustergrammer_widget/clustergrammer/__init__.py in cluster(self, dist_type, run_clustering, dendro, views, linkage_type, sim_mat, filter_sim, calc_cat_pval, run_enrichr, enrichrgram)
     97                                    calc_cat_pval=calc_cat_pval,
     98                                    run_enrichr=run_enrichr,
---> 99                                    enrichrgram=enrichrgram)
    100 
    101   def make_clust(self, dist_type='cosine', run_clustering=True,

~/anaconda2/envs/sd2py3/lib/python3.6/site-packages/clustergrammer_widget/clustergrammer/make_clust_fun.py in make_clust(net, dist_type, run_clustering, dendro, requested_views, linkage_type, sim_mat, filter_sim, calc_cat_pval, sim_mat_views, run_enrichr, enrichrgram)
     67 
     68   if sim_mat is not False:
---> 69     sim_net = make_sim_mat.main(net, inst_dm, which_sim, filter_sim, sim_mat_views)
     70 
     71     net.sim = {}

~/anaconda2/envs/sd2py3/lib/python3.6/site-packages/clustergrammer_widget/clustergrammer/make_sim_mat.py in main(net, inst_dm, which_sim, filter_sim, sim_mat_views)
     11 
     12     sim_dict[inst_rc] = dm_to_sim(inst_dm[inst_rc], make_squareform=True,
---> 13                              filter_sim=filter_sim)
     14 
     15   sim_net = {}

~/anaconda2/envs/sd2py3/lib/python3.6/site-packages/clustergrammer_widget/clustergrammer/make_sim_mat.py in dm_to_sim(inst_dm, make_squareform, filter_sim)
     51 
     52   if filter_sim > 0:
---> 53     filter_sim = adjust_filter_sim(inst_sim_mat, filter_sim)
     54     inst_sim_mat[ np.abs(inst_sim_mat) < filter_sim] = 0
     55 

~/anaconda2/envs/sd2py3/lib/python3.6/site-packages/clustergrammer_widget/clustergrammer/make_sim_mat.py in adjust_filter_sim(inst_dm, filter_sim, keep_top)
     69 
     70     inst_series = pd.Series(val_vect)
---> 71     inst_series.sort(ascending=False)
     72 
     73     sort_values = inst_series.values

~/.local/lib/python3.6/site-packages/pandas/core/generic.py in __getattr__(self, name)
   5065             if self._info_axis._can_hold_identifiers_and_holds_name(name):
   5066                 return self[name]
-> 5067             return object.__getattribute__(self, name)
   5068 
   5069     def __setattr__(self, name, value):

AttributeError: 'Series' object has no attribute 'sort'
```

**Comment by cornhundred** [comment]
Thanks for raising the issue. The issue appears to be that we're using a deprecated attribute (`sort()`) for Pandas Series (see https://stackoverflow.com/questions/44123874/dataframe-object-has-no-attribute-sort) - the new attribute is `sort_values()`. 

I'll be releasing a new version with this fix next week. I'll update this issue once fixed.

**Comment by cornhundred** [comment]
This bug has been fixed as of version 0.7.0
